### ISSUE TITLE
Add Data Sources Page to documentation site

### ISSUE THREAD

**Issue by nickw-CWF** [Canadian-Wildlife-Federation/Documentation#6]
@alexlcwf 

While we are figuring out a new contract with Foundry and development is on pause, I am slightly worried about having new data be released without the data sources available. Can we add a ``Data Sources`` section under ``CABD User Resources`` (maybe after ``Web Mapping Interface``)? I'll get Foundry to just switch out the hyperlink for the Data Sources button in the web tool for now.

Basically just put in the content of our data sources document in any format that is easiest/works best.

Also, do we have all the data sources update for BC?
